const/menu.ts

```diff
@@ -24,29 +24,29 @@ export const FOOTER_LINK_GROUPS: FooterLinkGroup[] = [
       { label: 'Analytics', url: url.analytics, type: 'external' },
       { label: 'Careers', url: '/careers' },
       { label: 'Grants', url: url.grants, type: 'external' },
-      { label: 'Explorer', url: url.explorer, type: 'external' },
+      { label: 'Explorer', url: url.explorer, type: 'external', utm: true },
```

Comment:
Does the Explorer know anything about UTM codes?

Reply:
It doesn't (kind of), Google Analytics does. 
If it is a UI we control is good to add it IMO